unittests/allplatformstests.py

```diff
@@ -4453,6 +4453,10 @@ def test_devenv(self):
         self.assertIn(f'TEST_C="{expected}"', o)
         self.assertIn('export TEST_C', o)
 
+        cmd = self.meson_command + ['devenv', '-C', self.builddir] + python_command + ['-c', 'import sys; sys.exit(42)']
```

Comment:
```suggestion
        cmd = self.meson_command + ['devenv', '-C', self.builddir] + python_command + ['-c', 'raise SystemExit(42)']
```

is slightly shorter, just for the record. :)